komodo/docker-compose.yml

```
@@ -0,0 +1,114 @@
+    environment:
+      DOCKER_ENSURE_BRIDGE: "dind0:10.32.0.1/16"
+    entrypoint: /entrypoint.sh
+    command: >
+      dockerd
+        --bridge dind0
+        --data-root /data/data
+        --exec-root /data/exec
+        --host unix:///data/docker.sock
+        --pidfile /data/docker.pid
+    volumes:
+      - ${APP_DATA_DIR}/entrypoint.sh:/entrypoint.sh
+      - ${APP_DATA_DIR}/data/docker:/data
+
+  postgres:
+    image: ghcr.io/ferretdb/postgres-documentdb:17-0.104.0@sha256:1d709fc03dcb82faa2e3aa02c0f80ea10e483b38aff08a05da36a6faa2098608
+    user: "1000:1000"
+    labels:
+      komodo.skip:
+    restart: on-failure
+    volumes:
+      - ${APP_DATA_DIR}/data/postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: admin
+      POSTGRES_PASSWORD: admin
+      POSTGRES_DB: postgres
+
+  ferretdb:
+    image: ghcr.io/ferretdb/ferretdb:2.3.1@sha256:1170bd36c39378614edd3f1d2b92f0c5a9b30e482a79ea0c3fc0fe11baba3afa
+    user: "1000:1000"
+    labels:
+      komodo.skip:
+    restart: on-failure
+    depends_on:
+      - postgres
+    volumes:
+      - ${APP_DATA_DIR}/data/ferretdb:/state
+    environment:
+      FERRETDB_POSTGRESQL_URL: postgres://admin:admin@postgres:5432/postgres
+
+  core:
+    image: ghcr.io/moghtech/komodo-core:1.18.4@sha256:b9a0c8757dd9e8f77a5e541c3afd66d37c5dbb1fa4a4687a2c6146741f18850e
+    user: "1000:1000"
+    labels:
+      komodo.skip:
+    restart: on-failure
+    depends_on:
+      - ferretdb
+    environment:
+      KOMODO_DATABASE_ADDRESS: ferretdb:27017
+      KOMODO_DATABASE_USERNAME: admin
+      KOMODO_DATABASE_PASSWORD: admin
+      COMPOSE_KOMODO_IMAGE_TAG: latest
+      KOMODO_PASSKEY: S3curePasskey2025!
+      TZ: Europe/Berlin
+      KOMODO_HOST: https://demo.komo.do
+      KOMODO_TITLE: Komodo
+      KOMODO_FIRST_SERVER: https://periphery:8120
+      KOMODO_DISABLE_CONFIRM_DIALOG: "false"
+      KOMODO_MONITORING_INTERVAL: "15-sec"
+      KOMODO_RESOURCE_POLL_INTERVAL: "1-hr"
+      KOMODO_WEBHOOK_SECRET: W3bh00kS3cret!2025
+      KOMODO_JWT_SECRET: JwtS3cretKey!2025
+      KOMODO_JWT_TTL: "1-day"
+      KOMODO_LOCAL_AUTH: "true"
+      KOMODO_DISABLE_USER_REGISTRATION: "false"
+      KOMODO_ENABLE_NEW_USERS: "false"
+      KOMODO_DISABLE_NON_ADMIN_CREATE: "false"
+      KOMODO_TRANSPARENT_MODE: "false"
+      KOMODO_LOGGING_PRETTY: "false"
+      KOMODO_PRETTY_STARTUP_CONFIG: "false"
+      KOMODO_OIDC_ENABLED: "false"
+      KOMODO_GITHUB_OAUTH_ENABLED: "false"
+      KOMODO_GOOGLE_OAUTH_ENABLED: "false"
+      #KOMODO_AWS_ACCESS_KEY_ID:
+      #KOMODO_AWS_SECRET_ACCESS_KEY:
+    volumes:
+      - ${APP_DATA_DIR}/data/core:/repo-cache
+      
+  periphery:
+    image: ghcr.io/moghtech/komodo-periphery:1.18.4@sha256:0a3ccd9294b13d3c5942d1e20392c81e28369ba1aaba6b17bfd5b35907760b98
+    user: "1000:1000"
+    labels:
+      komodo.skip:
+    restart: on-failure
+    environment:
+      COMPOSE_KOMODO_IMAGE_TAG: latest
+      KOMODO_PASSKEY: S3curePasskey2025!
```

Would use the provided password variable here.
```suggestion
      KOMODO_PASSKEY: ${APP_PASSWORD}
```